pkg/cli/admin/nodeimage/basenodeimagecommand.go

```diff
@@ -409,3 +441,37 @@ func (o *BaseNodeImageCommand) runNodeJoinerPod(ctx context.Context, tasks []fun
 	}
 	return nil
 }
+
+func (c *BaseNodeImageCommand) getIdmsFile(ctx context.Context) string {
+	imageDigestMirrorSets, idmsErr := c.ConfigClient.ConfigV1().ImageDigestMirrorSets().List(ctx, metav1.ListOptions{})
+	imageContentPolicies, icspErr := c.ConfigClient.ConfigV1().ImageContentPolicies().List(ctx, metav1.ListOptions{})
+	if idmsErr != nil && icspErr != nil {
+		return ""
```

Comment:
Only if `errors.IsNotFound()` for both the errors is true then it's ok to return an empty string, otherwise it may be a genuine error and should be reported as usual